app/views/layouts/application.html.erb

```diff
@@ -25,25 +25,9 @@
   <body class="mainstream <%= yield :body_classes %>">
     <div id="wrapper" class="<%= wrapper_class(@publication || @presenter) %>">
       <% unless current_page?(root_path) %>
-        <% if present_new_navigation? %>
-          <%=
-            render(
-              partial: 'govuk_component/beta_label',
-              locals: {
-                message: <<-MESSAGE
-                This is a test version of the layout of this page.
-                <a id=taxonomy-survey href='https://www.smartsurvey.co.uk/s/betasurvey2017?c=#{current_path_without_query_string}' target='_blank' rel='noopener noreferrer'>
-                  Take the survey to help us improve it
-                </a>
-                MESSAGE
-              }
-            )
-          %>
-        <% end %>
         <%= render partial: 'govuk_component/breadcrumbs',
           locals: {
-            breadcrumbs: breadcrumbs[:breadcrumbs],
-            collapse_on_mobile: present_new_navigation?
```

Comment:
Was a design decision made to actively remove this feature?

Reply:
It's still in the component, and we are considering whether it is useful everywhere (if it is we'd make it the default).

But we may also redesign aspects of the breadcrumb as part of our multivariate test.

This change is consistent with what we've done on government-frontend.

Reply:
This should still be collapsed on mobile when the taxonomy is shown. Good catch.